shell/packages/accounts-identity/accounts-identity-server.js

```diff
@@ -0,0 +1,198 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+Meteor.methods({
```

Comment:
I'm vaguely concerned about the operations implemented in these methods being non-transactional. It may be a good idea to go over each one and think carefully about what happens if other operations are occurring on the same user or identity at the same time, and whether there is any way to adjust the Mongo queries to avoid this. findAndModify() tends to be useful for this (combines a read and a write). For the cases that are non-trivial to fix, add a comment describing what could go wrong so we can think about it more later.